Makefile

```diff
@@ -13,9 +13,10 @@ test-install: install
 	. venv/bin/activate; pip install -r test/requirements.txt
 
 test: test-install
+	. venv/bin/activate; coverage run -m unittest discover -s test/unit
 
 test-integ: test
-	. venv/bin/activate; coverage run -m unittest discover
+	. venv/bin/activate; coverage run -m unittest discover -s test/integ
```

Comment:
Why does `test-integ` also run the unit tests? Shouldn't it only depend on `test-install`?

Reply:
Thought it would be nice to run both since the unit tests are pretty quick, plus the `docker-test` workflow calls `make test-integ` so it will run the unit tests too.